examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -89,6 +90,12 @@ bool sHaveBLEConnections                 = false;
 
 chip::DeviceLayer::DeviceInfoProviderImpl gExampleDeviceInfoProvider;
 
+DeferredAttribute gCurrentLevelPersister(ConcreteAttributePath(kLightEndpointId, Clusters::LevelControl::Id,
```

Comment:
Could we please add some comments, why we use deffered attribute persister and how it works (just 1-2 sentences)? This is great example for end users to take it and use it in their apps.